src/tests/test_iterative_improvement.py

```
@@ -0,0 +1,172 @@
+        print(f"\n📊 IMPROVEMENT RESULTS")
+        print("=" * 40)
+        
+        initial_return = improvement_results['iteration_results'][0]['performance']['total_return']
+        best_return = improvement_results['best_performance']['total_return']
+        improvement = improvement_results['improvement_achieved']
+        
+        print(f"Initial Return: {initial_return:.1f}%")
+        print(f"Best Return: {best_return:.1f}%")
+        print(f"Improvement: {improvement:+.1f}%")
+        
+        print(f"\n📈 Iteration Performance:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)